### ISSUE TITLE
New icons design

### ISSUE THREAD

**Issue by serge-rider** [dbeaver/dbeaver#2680]
Existing DBeaver icons are too 90s. 
And also they are taken from different freeware icon sets. This makes look-and-feel inconsistent and too speckle.
Let's try to make custom DBeaver icon set.

**Comment by nanny07** [comment]
Sincerely I don't like it: they are way too similar between them and they are difficult to distinguish.

Example: Editor SQL, Recent SQL Editor and New Editor SQL. They differ by the small bottom-right angle and it's hard to read at first sight (at least in my monitor/resolution)

**Comment by efradkin** [comment]
@nanny07 me too

**Comment by AndrewKhitrin** [comment]
@nanny07 @efradkin 

Anyway new set better than old one.

**Comment by efradkin** [comment]
It's to compare:
![image](https://user-images.githubusercontent.com/16664546/34376196-9a056da0-eafb-11e7-8568-a9ab0f60938f.png)

**Comment by serge-rider** [comment]
We just started to change icons, they are not final. Any feedback is very welcome.

@nanny07 agreed about SQL editor icons. They have to be not so similar. Will fix them in one of next versions.
@efradkin I also don't line blurring. Also going to fix this wherever it is possible.

**Comment by GiantCrocodile** [comment]
I see a general issue here. It is rather less about how the new icons look like. I don't like them much because the new icons don't fit to any other program I do use and I can't distinguish between the different icons. Maybe we should use some unified, simple icon set. Something that already exists and is proven in real world.

My main point is that I really enjoy DBeaver and it is one of the best open source softwares I can think of. Sadly, you do the same thing as Microsoft does with Office 365: DBeaver currently messes around with the things the user knows and loves. I don't want to update a software and see it completely changed over night. I use DBeaver quite often and I want to be productive. It's like with Microsoft Office 365 and their Teams, Outlook, buggy Office, etc.: You can't work with it because all time some update destroys some main feature, the design changes, you got new bugs or there are changes without a clear notification about what did change after starting the program. The user is left alone and has no clue what is wrong nor how to get it back. I noticed this in past of DBeaver and I think this is something we hsould completely avoid in any scenario. Massive changes to DBeaver should be not the default setting. Rather the user should be able to switch it in the options or on start up by pressing a yes/no button in some 'Do you wanna change your theme? YES/NO' dialogue. If I know that there is some new theme/icon set or if I get told so, I want to decide when it gets tested. It shouldn't get in my way of my daily routine.

Long story short: We need to keep more care of not disturbing users by updates and thus increase our productivity like with previous updates :)..

**Comment by serge-rider** [comment]
@GiantCrocodile Generally I agree with your main point - we shouldn't make too radical changes in UI.
In regards to icons - it'd be great to have switchable icon sets so user could decide which set he likes more. But it is quite difficult (let's say - impossible) to support this on platform level.

And we can't keep old rusty icons from different icon sets forever. It doesn't look solid and it doesn't look professional.

If you have any suggestion on particular icons - please tell. We have a designer who can draw any custom icons.

**Comment by GiantCrocodile** [comment]
To me it is unclear why some icons are orange, some dark blue and some light blue or mixed. The colors should be same or the meaning of the color should be clear. Blue and orange doesn't fit together. Also some icons may need a deeper detail level. Like less tiny and more spacing to show what you mean. Another idea is that we could use something like Font Awesome Icons and add some icons we miss in that pack with same kind of style. I'm no designer nor do I really know basics about design but I can rate what I see as an user.

One other thing I notice is that these icons don't fit to the borders, buttons, window design. It looks like the icons are taken from program A and the other parts of GUI are taken from program B.

What was wrong about the previous icons? They didn't feel like 90s to me nor anyway outdated. There are way worse examples of 90s looking icons/programs than DBeaver.

**Comment by serge-rider** [comment]
> To me it is unclear why some icons are orange, some dark blue and some light blue or mixed.
Generally most of icons are blue or light blue. 
Orange icons are for folders. Also orange/green is used as a second color for some action icons.
I see that a few icons are not fit (e.g. connect/disconnect and SQL editor open icons) and they will be re-done.

And yes, most of previous icons were actually made in 90s or in early 00s (e.g. Silk icon set).
They are good icons but nowadays when we have flat design everywhere - they become really rusty.

**Comment by benyanke** [comment]
Just popping in to say I love the new icons. A nice surprise after upgrade the other day.

**Comment by Yanpas** [comment]
Is there a way to disable new icons? They are nicer, but:
1. Not familiar to me
2. Looks odd with old-one icons in Eclipse (I use plugin). Too eclectic
![image](https://user-images.githubusercontent.com/7345761/34899787-18603aa0-f80c-11e7-92e3-9a7f09e38431.png)

3. They are too similar to each other

**Comment by serge-rider** [comment]
@Yanpas There is no way to disable new icons. Eclipse doesn't support "icon themes".
We going to make them less eclectic and less similar in next versions.

BTW you could help if you would say what icons are the worst ;) (connect and SQL editor icons are already fixed)

**Comment by Otiel** [comment]
I preferred the old icons as well, I think they were way more easy to read. I understand you want to change since they're based on a 2000s icon set like Silk, but for 16x16 icons, I think that this is the style that works best, not flat icons.

Do you know the [Fugue set](http://p.yusukekamiyamane.com/)? It's the same style as the old DBeaver icons, but a bit fresher.
![fugue preview](http://p.yusukekamiyamane.com/icons/preview/fugue.png)

Anyway, if you choose to continue with the new icons, the main issue is that almost all icons are blurry.

I'm also not a fan of the tower/cell design for the connection icon. I think the database icon was a better choice.

**Comment by serge-rider** [comment]
Generally most icons were updated (blurry ones fixed, connection icon changed)
Any further improvements should be tracked in separate tickets.
